src/Swarm/Language/Directions.hs

```diff
@@ -0,0 +1,116 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+--
+-- Abstract syntax for terms of the Swarm programming language.
+module Swarm.Language.Directions (
```

Comment:
It's not a big deal, but I think we have generally settled on the convention of module names being singular.